scripts/compare_noupdate_xml_records.py

```diff
@@ -211,5 +181,68 @@ def main(argv=None):
         document, pretty_print=True, xml_declaration=True, encoding='utf-8')
 
 
+def main(argv=None):
+    """
+    Attempt to represent the differences in data records flagged with
+    'noupdate' between two different versions of the same Odoo module or
+    repository.
+
+    Print out a complete XML data file that can be loaded in a post-migration
+    script using openupgrade::load_xml().
+
+        :param argv: arg1 (old) and arg2 (new) are the module o repository
+        path, and arg3 (mode) are the 'module' or 'repository' options.
+
+    Known issues:
+    - Does not detect if a deleted value belongs to a field
+      which has been removed.
+    - Ignores forcecreate=False. This hardly occurs, but you should
+      check manually for new data records with this tag. Note that
+      'True' is the default value for data elements without this tag.
+    - Does not take csv data into account (obviously)
+    - Is not able to check cross module data
+    - etree's pretty_print is not *that* pretty
+    - Does not take translations into account (e.g. in the case of
+      email templates)
+    - Does not handle the shorthand records <menu>, <act_window> etc.,
+      although that could be done using the same expansion logic as
+      is used in their parsers in openerp/tools/convert.py
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'olddir', metavar='older_directory')
+    parser.add_argument(
+        'newdir', metavar='newer_directory')
+    parser.add_argument(
```

Comment:
Can you make this argument optional and default to `module` for having the same command line as previously?